specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json

```diff
@@ -6907,7 +6907,7 @@
           "type": "integer",
           "format": "int32",
           "minimum": 0,
-          "maximum": 2,
+          "maximum": 4,
```

Comment:
Unless we are confident that this value will never ever go up in the future, please remove the `maximum` attribute.